Formula/ripgrep.rb

```diff
@@ -19,6 +19,14 @@ def install
 
     bin.install "target/release/rg"
     man1.install "doc/rg.1"
+
+    # Completion scripts are generated in the crate's build directory, which
+    # includes a fingerprint hash. Try to locate it first
+    out_dir = Dir["target/release/build/ripgrep-*/out"].first
+    if out_dir
```

Comment:
You can let it fail; the stacktrace will indicate the error as well as its location.